src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -88,7 +88,7 @@ foam.CLASS({
   `,
 
   messages: [
-    { name: 'REFRESH_MSG', message: 'Table Refreshed' }
+    { name: 'REFRESH_MSG', message: 'Refresh Requested. Please be patient, refresh may take up to a minute.' }
```

Comment:
```suggestion
    { name: 'REFRESH_MSG', message: 'Refresh Requested ... ' }
```

Reply:
Current message has to be updated to reflect an in progress operation .. we are trying to fix the minute long intervals but in the mean time, the message *has* to be updated.